### PR TITLE
added initial git structure

### PR DESCRIPTION
**Pull request by MartinSchmidt**
#minor

### CODE REVIEW

File: git/conventions.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### /docs
+
+This folder should contain all .md files, pictures
+and other files that are used to document the repository.
+
+### /chart
+
+This is the folder for all our repos containing a helm chart.
+The chart will always be in the chart folder, which may
+contain 3 folders
```
Comment (jacobcrawford) [inline git/conventions.md:52]: Use ":" consistently before listing.

File: git/conventions.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- ./Chart.yaml
+- ./values.yaml
+- ./values.schema.json
+
+The values.schema.json is used to validate the value input 
+files, and should always ensure correct use of the chart, 
+more info can be found here: https://helm.sh/docs/topics/charts/
+
+### /.artifacts
```
Comment (jacobcrawford) [inline git/conventions.md:68]: What is the reasoning behind making the artifacts folder hidden?

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
Small things. Nothing much. What should a person even write here?

File: git/conventions.md (hunk 1 above)
Comment (MartinSchmidt) [reply git/conventions.md]: I have fixed the issue and updated the PR

File: git/conventions.md (hunk 2 above)
Comment (MartinSchmidt) [reply git/conventions.md]: You are right, was a typo from my side.

**Review by jaxels10 — APPROVED** [review]
I've read through it and I think it looks good.

**Review by jacobcrawford — APPROVED** [review]

**Pull request merged**